installer/common/install-cloudflow.sh

```diff
@@ -14,39 +14,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#Usage: gke-install-cloudflow <CLUSTER_NAME> <CLOUDFLOW_NAMESPACE>
-
+# Usage: install-cloudflow.sh [CLUSTER_NAME] [CLOUDFLOW_NAMESPACE] [CLUSTER_TYPE]
+if [ $# -ne 2 ]; then
```

Comment:
this needs to be 3.

Reply:
> yes a protection against a null value is required. I will do another round now for the PR.

done

> You need to set to 3 the number of args ;)

and done!

Should be ready to go 🙏